web/modules/custom/bnf/src/Services/ImportContextStack.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\bnf\Services;
+
+use Drupal\bnf\ImportContext;
+
+/**
+ * Manages the current import context.
```

Comment:
I know what it does, and you know what it does, but for future readers, this will mean nothing.

Can you expand the documentation?